tests/integration/ceph_base_object_test.go

```diff
@@ -199,6 +201,13 @@ func testObjectStoreOperations(s suite.Suite, helper *clients.TestClient, k8sh *
 				time.Sleep(5 * time.Second)
 				continue
 			}
+			logger.Info("objectstore status is", objectStore.Status)
+			if objectStore.Status.BucketStatus.Health == cephv1.ConditionFailure && CephObjectStoreCIRetryCount == 0 {
+				CephObjectStoreCIRetryCount = CephObjectStoreCIRetryCount + 1
+				createCephObjectStore(s, helper, k8sh, namespace, storeName, 3, false)
```

Comment:
Why are we again creating the object store here?

Reply:
I was thinking to retry the whole process of the test.
Maybe something goes wrong with the `object store` status while creating it.
Tried the other way by increasing the [wait time](https://github.com/rook/rook/pull/8413#discussion_r684369787)